docs/content/latest/deploy/multi-dc/async-replication.md

```diff
@@ -102,6 +102,23 @@ Once you have set up replication, load data into the source universe as follows:
 
 When completed, proceed to [Verifying Replication](#verifying-replication).
 
+## Setting Up Replication with TLS
+
+### Source and Target Universes have the Same Certificates
+
+If both universes use the same certificates, run the same `yb-admin` [`setup_universe_replication`](../../../admin/yb-admin/#setup-universe-replication) command as above, but also include the [`-certs_dir_name`](../../../admin/yb-admin#syntax) flag. Setting that to the target universe's certificate directory will make replication use those certificates for connecting to both universes.
+
+### Source and Target Universes have Different Certificates
+
+When both universes use different certificates, we need to store the certificates for the producer universe on the target universe. Each node on the target universe will have a parent directory (determined by the `certs_for_cdc_dir` gflag), under which there will be sub-directories for each source universe, which each contain their respective source universe's certificates.
```

Comment:
"Each node on the target universe will have a parent directory (determined by the `certs_for_cdc_dir` gflag), under which there will be sub-directories for each source universe, which each contain their respective source universe's certificates." sentence should be broken down into bite-size chunks.